doc/DeveloperGuide.md

```diff
@@ -57,6 +57,8 @@ Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
 `* * *` | user | add a new person |
+`* * *` | user | edit a person | edit an existing person in addressbook with new information
+`* * *` | user | find a person by tag | find a person by locating the tags
```

Comment:
your "I want to" and "So that I can" are saying the same thing. Please see how you can improve this.